owncloud/android
# [New Arch] File Details Fragment

Refactor FileDetailsFragment to add reactivity. The file is observed from the database via Flow, so we can update the details as soon as the database is updated with any changes.
_____

## QA

Check list: https://github.com/owncloud/android/pull/3742#issuecomment-1247972457

- [x] (1) [P1] Bad UI when download is interrupted https://github.com/owncloud/android/pull/3742#issuecomment-1247994324
- [x] (2) [P2] Thumbnail repeated action https://github.com/owncloud/android/pull/3742#issuecomment-1248001277 [FIXED]
- [x] (3) [P1] Available offline behaviour in Details https://github.com/owncloud/android/pull/3742#issuecomment-1248001277 [FIXED]
- [x] (4) Crash after removing file in Details https://github.com/owncloud/android/pull/3742#issuecomment-1248072598 [FIXED]
